criu/mount-v2.c

```diff
@@ -950,7 +954,7 @@ static int restore_one_sharing(struct sharing_group *sg, struct mount_info *targ
 			if (move_mount_set_group(first->mnt_fd_id, NULL, target->mnt_fd_id)) {
 				pr_err("Failed to copy sharing from %d to %d\n", first->mnt_id, target->mnt_id);
 				close(target_fd);
-				return -1;
+				goto err;
```

Comment:
If we `close_safe(&target_fd)` in `err` label, maybe we can remove `close(target_fd)` everywhere before `goto err`?

Reply:
You are right. Done. Thanks.